src/main/java/io/cdap/plugin/http/source/common/BaseHttpSourceConfig.java

```diff
@@ -870,6 +873,9 @@ public static Long toLong(String value, String propertyName) {
     String[] mappings = keyValueString.split(",");
     for (String map : mappings) {
       String[] columns = map.split(":");
+      if (columns.length < 2) {
```

Comment:
Is this error seen during configure time? If yes, you should add the failure to the failure collector

Reply:
created https://cdap.atlassian.net/browse/PLUGIN-1109 for the same